android/app/src/main/java/org/uwaterloo/subletr/pages/home/list/HomeListChildView.kt

```diff
@@ -113,43 +113,89 @@ fun HomeListChildView(
 
 		LaunchedEffect(key1 = lastItemIsShowing) {
 			if (lastItemIsShowing) {
-				viewModel.listingPagingParamsStream.onNext(
-					HomeListChildViewModel.ListingPagingParams(
-						previousListingItemsModel = uiState.listingItems,
-						pageNumber = (
-							listState.layoutInfo.totalItemsCount
-							).floorDiv(HomeListChildViewModel.LISTING_PAGE_SIZE)
+				viewModel.getListingParamsStream.onNext(
+					HomePageViewModel.GetListingParams(
+						filters = uiState.filters,
+						transportationMethod = HomePageUiState.TransportationMethod.ALL,
+						listingPagingParams = HomePageViewModel.ListingPagingParams(
+								previousListingItemsModel = uiState.listingItems,
+								pageNumber = listState.layoutInfo.totalItemsCount
+									.floorDiv(HomePageViewModel.LISTING_PAGE_SIZE)
+						),
```

Comment:
yay pagination

Reply:
YAY, pagination definitely makes me super happy about life and all /s